.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - run: wget https://s3.amazonaws.com/testrpc-shapshots/${CONTRACTS_COMMIT_HASH}.zip
       - run: unzip ${CONTRACTS_COMMIT_HASH}.zip -d testrpc_snapshot
       - run: node ./node_modules/lerna/bin/lerna.js bootstrap
-      - run: yarn lerna:run bootstrap
```

Comment:
I agree that bootstrap should be removed but most if not all packages build as a part of their test:circleci commands